package org.json;

import kotlin.Metadata;

/* JADX INFO: loaded from: classes3.dex */
@Metadata(d1 = {"\u0000\u001c\n\u0002\u0018\u0002\n\u0002\u0010\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0010\t\n\u0000\n\u0002\u0010\u0002\n\u0002\b\u0002\bf\u0018\u00002\u00020\u0001J\u001a\u0010\u0007\u001a\u00020\u00062\u0006\u0010\u0003\u001a\u00020\u00022\b\b\u0002\u0010\u0005\u001a\u00020\u0004H&J\u0010\u0010\u0007\u001a\u00020\u00062\u0006\u0010\u0003\u001a\u00020\u0002H&ø\u0001\u0000\u0082\u0002\u0006\n\u0004\b!0\u0001¨\u0006\bÀ\u0006\u0001"}, d2 = {"Lcom/ironsource/ot;", "", "Ljava/lang/Runnable;", "action", "", "delay", "", "a", "mediationsdk_release"}, k = 1, mv = {1, 8, 0})
public interface ot {

    /* JADX INFO: renamed from: com.ironsource.ot$-CC, reason: invalid class name */
    public final /* synthetic */ class CC {
        public static /* synthetic */ void a(ot otVar, Runnable runnable, long j, int i, Object obj) {
            if (obj != null) {
                throw new UnsupportedOperationException("Super calls with default arguments not supported in this target, function: postReleaseTask");
            }
            if ((i & 2) != 0) {
                j = 0;
            }
            otVar.a(runnable, j);
        }
    }

    void a(Runnable action);

    void a(Runnable action, long delay);
}
